projects/plugins/protect/src/js/components/footer/index.jsx

```diff
@@ -4,13 +4,14 @@
 import React from 'react';
 import { Container, Col, Text, Button, Title, IconsCard } from '@automattic/jetpack-components';
 import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
 
 /**
  * Internal dependencies
  */
 import styles from './styles.module.scss';
 
-const Footer = () => {
+const Footer = ( { handleProductButton, learnMoreUrl } ) => {
```

Comment:
Actually... I wonder whether we can make this component self sufficient, moving the checkout logic straight here. We can remove the props, too. That's a benefit about using react hooks.

Reply:
I tried to leave this component as a pure layout component as possible... to be easier to extract it if we eventually decide to move it com js-packages/components as one reusable element